### ISSUE TITLE
Double jump sound doesn't play

### ISSUE THREAD

**Issue by blipdrifter** [youkaicountry/yknytt#179]
The extra high-pitched jumping sound that normally plays when you double jump in KS is not present in YKnytt as of 0.5.1.

**Comment by up-left** [comment]
tbh I just used to playing without double jump sound. so I have no plans to add it.

**Comment by djsolidsnake86** [comment]
> patch confirmed working :), thanks!

where is the patch for the double jump sound?

**Comment by jakerosado** [comment]
Would you be willing to accept my fixes for open issues @up-left?

**Comment by up-left** [comment]
I'm suspicious about that. Only if they contain real value and not personal preference.

**Comment by jakerosado** [comment]
Yup, no personal preferences from me. Just resolving open issues like this one.

**Comment by up-left** [comment]
Unfortunately, open issues are in a mess right now, most of them have a low result to labor ratio. Maybe it's possible to find a better upscaler, catch a left/right bug by ms2048, or switch to a sqlite (massive task). So I'm confused about which issues you can fix here.
